addons/hivedr/src/main/java/org/apache/falcon/hive/HiveDRTool.java

```diff
@@ -271,8 +271,8 @@ private void cleanStagingDirectory() throws HiveReplicationException {
     private String sourceEvents() throws Exception {
         MetaStoreEventSourcer defaultSourcer = null;
         String inputFilename = null;
-        String lastEventsIdFile = FileUtils.DEFAULT_EVENT_STORE_PATH +File.separator+inputOptions.getJobName()+"/"
-                +inputOptions.getJobName()+".id";
+        String lastEventsIdFile = FileUtils.DEFAULT_EVENT_STORE_PATH + File.separator
+                + inputOptions.getJobName() + "/" + inputOptions.getJobName() + ".id";
```

Comment:
Looks like a format issue but no code change? You can use File.separator instead of "/"